### DISCUSSION TITLE
What features arent available in TE3

### DISCUSSION THREAD

**Discussion by capstoneanalytics** [TabularEditor/TabularEditor3#747] [Other Topics]
Hello,

Im evaluating TE3 for a client and so far what im seeing is excellent for enterprise model building. I just wanted to know what features arent available in TE3 and are only available in PBI desktop. Something like Q&A for example, I could not see any documentation that you can set it up with TE3. Is there anything else ?

Abhijith

**Comment by otykier** [comment]
Q&A is a visual. Tabular Editor doesn’t deal with the visual aspect of building reports - only the data model.

With regards to the data model, there are no limitations except when TE is used as an [external tool to Power BI Desktop](https://learn.microsoft.com/en-us/power-bi/transform-model/desktop-external-tools#data-modeling-operations).

Moreover, Tabular Editor doesn’t have the Power Query editor. You can still edit M expressions, though.

**Reply by capstoneanalytics** [reply]
Thanks Daniel. I should have clarified regarding Q&A. You can set up Q&A in PBI desktop where you can define synonyms, show/hide fields to be used in Q&A, teach Q&A  etc. Im guessing this is not supported in TE3 ?

**Reply by otykier** [reply]
Technically, you can still edit the synonyms within TE as well. There’s a property on the Culture object called [LinguisticMetadata](https://learn.microsoft.com/en-us/dotnet/api/microsoft.analysisservices.tabular.linguisticmetadata?view=analysisservices-dotnet), which is exposed in TE. However, the content is raw JSON/XML. We’d like to provide a better editing experience for this data, however Microsoft has not documented the linguistic schema format.

**Reply by otykier** [reply]
You can find more details in [this thread](https://github.com/TabularEditor/TabularEditor/issues/257) (applies to both TE2 and TE3).

**Reply by capstoneanalytics** [reply]
Thanks Daniel
